packages/cli-template-contracts-hardhat/package.json

```diff
@@ -32,7 +32,7 @@
         "test:report-gas": "REPORT_GAS=true hardhat test",
```

Comment:
I think `.yarn` and `.yarnrc.yml` can also be removed from the files list.

Reply:
The project is using `yarn` and if these files are not downloaded, devs would need to at least create a `.yarnrc.yml` file. We could remove them when we adapt the templates to use other package manager by default like `npm` for example. What do you think?

For this specific template we could remove the `.yarn` folder and modify the `yarnrc.yml` file to have only `nodeLinker: node-modules`.

Reply:
Sorry, I meant from the package.json`s `files` field list.